docs/source/ProgrammingGuide/View.rst

```diff
@@ -94,17 +94,17 @@ The requirement that the destructor of `T` not deallocate memory technically dis
 
 Finally, note that virtual functions are technically allowed, but calling them is subject to further restrictions; developers should consult the discussions in Chapter 13, Kokkos and Virtual Functions (under development).
 
-6.2.3 Can I make a View of Views?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Can I make a View of Views?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. warning::
 
   NOVICES: THE ANSWER FOR YOU IS "NO."  PLEASE SKIP THIS SECTION.  
 
 A "View of Views" is a special case of View, where the type of each entry is itself a View. It is possible to make this, but before you try, please see below.
 
-6.2.3.1 You probably don't want this
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You probably don't want this
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This one had the wrong underlining.

Reply:
![image](https://github.com/user-attachments/assets/2d03a7cc-bf43-4949-a93e-28f69d4552a2)